src/conversion/downcastdispatcher.js

```diff
@@ -126,23 +129,26 @@ export default class DowncastDispatcher {
 	 * Takes {@link module:engine/model/differ~Differ model differ} object with buffered changes and fires conversion basing on it.
 	 *
 	 * @param {module:engine/model/differ~Differ} differ Differ object with buffered changes.
+	 * @param {module:engine/view/writer~Writer} writer View writer that should be used to modify view document.
 	 */
-	convertChanges( differ ) {
+	convertChanges( differ, writer ) {
+		this.conversionApi.writer = writer;
```

Comment:
Do we need to set it up here if it is set up in every method called later?

BTW. I don't have a better solution for now, but it seems kind of awkward how this writer is obtained and then passed down to deeper methods in dispatcher. I am sure that you thought of different ways of solving that so I don't necessarily want to start a discussion, just leaving it here as a thought.

Reply:
I got used to it during the work on it, but it may look a little awkward. I couldn't find better solution either. Editing controller is listening for changes in the document and then starts to converting changes in `view.change()` block, passing writer down to the dispatcher and from there to the `conversionApi` which can be used by converters.